### ISSUE TITLE
[request] Move (days since 2000) to (current ( v)) block

### ISSUE THREAD

**Issue by kyleplo** [LLK/scratch-gui#1710]
Since the `(days since 2000)` block isn't that popular or useful, I think putting it in with `(current ( v))` is a better idea:
![image](https://user-images.githubusercontent.com/31634240/38208620-a3da206c-367f-11e8-97c0-724ae4f14437.png)

**Comment by hedgehog125** [comment]
Yes, however, it does still have some good uses. For instance you can use it to get UTC time and then you can use it to find out which time zone the user is in. (or their clock is set to) Maybe there could be some more options so you can easily find out the UTC time and their time zone?

**Comment by kyleplo** [comment]
@hedgehog125 Those are complex features that not everyone needs; I'm not proposing the removal, I'm proposing moving it.

**Comment by hedgehog125** [comment]
@kyleplo Yes, I just thought it could be a useful feature while it was being discussed.
